### PR TITLE
Initial Marketplace docs

### PR DESCRIPTION
**Pull request by escopecz**
User docs for https://github.com/mautic/mautic/pull/9650/

### CODE REVIEW

File: pages/28.Marketplace/default.en.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+The context menu shows actions you can take.
+- Close will take you to the List View
+- Install will install the plugin
+- Issue Tracker will open a new window with the issue tracker. It will show up only if the plugin has this information available. Use this option to search for issues with the pluin and report new ones.
+
+![Marketplace detail](marketplace-detail.png)
+
+### CLI command
+
+The marketplace has commands that can be handy for those who prefer CLI over UI or for some automations.
```
Comment (RCheesley) [inline pages/28.Marketplace/default.en.md:74]:
```suggestion
The Marketplace has commands that can be used by those who prefer using the command line to the user interface, or for some automations.
```

File: pages/28.Marketplace/default.en.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+#### List plugins
```
Comment (RCheesley) [inline pages/28.Marketplace/default.en.md:76]:
```suggestion
#### List Plugins
```

File: pages/28.Marketplace/default.en.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+
+`bin/console mautic:marketplace:list` will list first page of available plugins like so:
```
Comment (RCheesley) [inline pages/28.Marketplace/default.en.md:78]:
```suggestion
`bin/console mautic:marketplace:list` will list first page of available Plugins like so:
```

File: pages/28.Marketplace/default.en.md [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+
+```
++-------------------------------------------------------+-----------+--------+
+| name                                                  | downloads | favers |
```
Comment (RCheesley) [inline pages/28.Marketplace/default.en.md:82]: Github refers to the count of folk having starred a repo as 'stargazers' - favers did not make sense really. Could also call it 'likes' but that does not correlate too well with GH?

```suggestion
| name                                                  | downloads | stargazers |
```

File: pages/28.Marketplace/default.en.md [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+| kuzmany/mautic-recommender-bundle                     | 250       | 30     |
+| kuzmany/mautic-custom-tags-bundle                     | 119       | 20     |
+| dazzle/mautic-sendinblue-bundle                       | 73        | 5      |
+|     Allows to send E-mails with Sendinblue            |           |        |
++-------------------------------------------------------+-----------+--------+
+Total packages: 69
+Execution time: 388 ms
+```
+
+There are options that will allow you to filter or go to next pages. The full list will show up when you add `--help` after the command as you are used to in other Mautic commands.
```
Comment (RCheesley) [inline pages/28.Marketplace/default.en.md:117]:
```suggestion
There are options that will allow you to filter or go to next pages. The full list will show up when you add `--help` after the command, as used in other Mautic commands.
```

File: pages/28.Marketplace/default.en.md [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+
+```
+  -p, --page[=PAGE]      Page number [default: 1]
+  -l, --limit[=LIMIT]    Packages per page [default: 15]
+  -f, --filter[=FILTER]  Filter the packages [default: ""]
+  -h, --help             Display this help message
+```
+
+Example usage how to search for Captcha plugin: `bin/console mautic:marketplace:list --filter=captcha`
```
Comment (RCheesley) [inline pages/28.Marketplace/default.en.md:126]:
```suggestion
Example usage how to search for Captcha Plugin: `bin/console mautic:marketplace:list --filter=captcha`
```

File: pages/28.Marketplace/default.en.md [hunk 7]
```diff
@@ -0,0 +1,138 @@
...
+
+### Planned Features
```
Comment (RCheesley) [inline pages/28.Marketplace/default.en.md:128]:
```suggestion
### Planned features
```

File: pages/28.Marketplace/default.en.md [hunk 8]
```diff
@@ -0,0 +1,138 @@
...
+
+The Marketplace is lacking some important features. In the text above there are mentions of several planned features in cursiva. Most of the features are waiting for Mautic 4 release and it's main features (Composer v2 support, new directory structure) and therefore can be developed when all these features will come together.
```
Comment (RCheesley) [inline pages/28.Marketplace/default.en.md:130]:
```suggestion
The Marketplace is lacking some important features. In the text above there are mentions of several planned features. Most of the features are waiting for the Mautic 4 release and its main features (Composer v2 support, new directory structure) and therefore can be developed when all these features come together.
```

File: pages/28.Marketplace/default.en.md [hunk 9]
```diff
@@ -0,0 +1,138 @@
...
+
+- Mautic version compatibility check. The Marketplace should tell you whether a plugin is compatible with your Mautic installation.
```
Comment (RCheesley) [inline pages/28.Marketplace/default.en.md:132]:
```suggestion
- Mautic version compatibility check. The Marketplace should tell you whether a Plugin is compatible with your Mautic installation,
```

File: pages/28.Marketplace/default.en.md [hunk 10]
```diff
@@ -0,0 +1,138 @@
...
+- Installation via CLI and UI. 
+- Update of installed plugins.
```
Comment (RCheesley) [inline pages/28.Marketplace/default.en.md:134]:
```suggestion
- Update of installed Plugins,
```

File: pages/28.Marketplace/default.en.md [hunk 11]
```diff
@@ -0,0 +1,138 @@
...
+- Automatic plugin updates. There will be configuration that will allow you to set globally whether you want to automatically upgrade plugins and also posibility to set that on per plugin bases. Automatic upgrades make sense only for bug fix releases. Other releases are too risky and should be handled manually. [API reference](https://packagist.org/apidoc#track-package-updates)
```
Comment (RCheesley) [inline pages/28.Marketplace/default.en.md:135]:
```suggestion
- Automatic Plugin updates - there will be configuration that will allow you to set globally whether you want to automatically upgrade Plugins and also have the possibility of configuring this at the plugin level. Automatic upgrades make sense only for bug fix releases. Other releases are too risky and should be handled manually. [API reference][track-updates]
```

File: pages/28.Marketplace/default.en.md [hunk 12]
```diff
@@ -0,0 +1,138 @@
...
+- List security advisories. [API reference](https://packagist.org/apidoc#list-security-advisories)
+- Notifications about new versions and found security vulnerabilities.
```
Comment (RCheesley) [inline pages/28.Marketplace/default.en.md:137]:
```suggestion
- Notifications about new versions and security vulnerabilities that have been identified,
```

File: pages/28.Marketplace/default.en.md [hunk 13]
```diff
@@ -0,0 +1,138 @@
...
+```
+
+Example usage how to search for Captcha plugin: `bin/console mautic:marketplace:list --filter=captcha`
+
+### Planned Features
+
+The Marketplace is lacking some important features. In the text above there are mentions of several planned features in cursiva. Most of the features are waiting for Mautic 4 release and it's main features (Composer v2 support, new directory structure) and therefore can be developed when all these features will come together.
+
+- Mautic version compatibility check. The Marketplace should tell you whether a plugin is compatible with your Mautic installation.
+- Installation via CLI and UI.
```
Comment (RCheesley) [inline pages/28.Marketplace/default.en.md:133]:
```suggestion
- Installation via CLI and UI,
```

File: pages/28.Marketplace/default.en.md [hunk 14]
```diff
@@ -0,0 +1,138 @@
...
+
+### Planned Features
+
+The Marketplace is lacking some important features. In the text above there are mentions of several planned features in cursiva. Most of the features are waiting for Mautic 4 release and it's main features (Composer v2 support, new directory structure) and therefore can be developed when all these features will come together.
+
+- Mautic version compatibility check. The Marketplace should tell you whether a plugin is compatible with your Mautic installation.
+- Installation via CLI and UI. 
+- Update of installed plugins.
+- Automatic plugin updates. There will be configuration that will allow you to set globally whether you want to automatically upgrade plugins and also posibility to set that on per plugin bases. Automatic upgrades make sense only for bug fix releases. Other releases are too risky and should be handled manually. [API reference](https://packagist.org/apidoc#track-package-updates)
+- List security advisories. [API reference](https://packagist.org/apidoc#list-security-advisories)
```
Comment (RCheesley) [inline pages/28.Marketplace/default.en.md:136]:
```suggestion
- List security advisories [API reference][advisories],
```

**Review by RCheesley — CHANGES_REQUESTED** [review]
Thanks for writing the docs page @escopecz !

I have made some suggestions on grammar and refactored the links to use the link macros that we use in the docs (see [Style Guide](https://docs.mautic.org/en/contributing/style-guide#links)).

Have also fixed some inconsistency in capitalisation and a couple of terms I have made suggestions for.

File: pages/28.Marketplace/default.en.md [hunk 15]
```diff
@@ -0,0 +1,138 @@
...
+articleenabled: false
+orgaenabled: false
+orga:
+    ratingValue: 2.5
+orgaratingenabled: false
+personenabled: false
+facebookenable: true
+---
+
+*Mautic 4 comes with new Marketplace. In the initial release there is just plugin browser. The install an update features will be added in the future feature releases as it depends on the new Composer v2 support and new file structure.*
```
[the remark this answers is not in the document]
Comment (escopecz) [reply pages/28.Marketplace/default.en.md:15]: Are the 3 dashes in the suggested line 15 required? They are already in the current line 13.

File: pages/28.Marketplace/default.en.md [hunk 16]
```diff
@@ -0,0 +1,138 @@
...
+
+**Warning**
+
+*The first marketplace version does not check for Mautic version compatibility yet as it requires change in each existing plugin, so please do not blindly manually install plugins you see there. Always check youself if they support your Mautic version.*
+
+### List of plugins
+
+The list of plugins is accessible from your Mautic administration menu. Click the cog icon in the top right hand corner to display the admin menu.
+
+The list view allows you to search for specific keywords. It displays quick stats of how many times a plugin was downloaded and how many stars it has in [Packagist](https://packagist.org). It also shows the vendor who has developed the plugin. Sadly, the sorting by columns is not available at the moment because it's not supported by the Packagist API.
```
[the remark this answers is not in the document]
Comment (escopecz) [reply pages/28.Marketplace/default.en.md:27]: How does the Markdown parser know what `[packagist]` link means?

File: pages/28.Marketplace/default.en.md (hunk 4 above)
Comment (escopecz) [reply pages/28.Marketplace/default.en.md:84]: favers is the original term used by Packagist. Do we want to rename it?

File: pages/28.Marketplace/default.en.md (hunk 15 above)
[the remark this answers is not in the document]
Comment (RCheesley) [reply pages/28.Marketplace/default.en.md:15]: Sorry pressed the wrong button! 

Yes they are required to add the horizontal line after the title when using the docs layout.

File: pages/28.Marketplace/default.en.md (hunk 16 above)
[the remark this answers is not in the document]
Comment (RCheesley) [reply pages/28.Marketplace/default.en.md:27]: This is a standard feature of Markdown using [link macros](https://www.markdownguide.org/basic-syntax/#reference-style-links). I added the macros at the bottom of the page.

File: pages/28.Marketplace/default.en.md (hunk 4 above)
Comment (RCheesley) [reply pages/28.Marketplace/default.en.md:84]: Ah OK - it just doesn't seem to make sense to me, like, it's not really a word. If that is what they call people who have starred a repo I guess we keep it?

File: pages/28.Marketplace/default.en.md [hunk 17]
```diff
@@ -0,0 +1,144 @@
...
+```
+
+Example usage how to search for Captcha Plugin: `bin/console mautic:marketplace:list --filter=captcha`
+
+### Planned features
+
+The Marketplace is lacking some important features. In the text above there are mentions of several planned features. Most of the features are waiting for the Mautic 4 release and its main features (Composer v2 support, new directory structure) and therefore can be developed when all these features come together.
+
+- Mautic version compatibility check. The Marketplace should tell you whether a Plugin is compatible with your Mautic installation,
+- Installation via CLI and UI,
```
Comment (RCheesley) [inline pages/28.Marketplace/default.en.md:139]: Suggestion from Vale linter

```suggestion
- Installation via command line and user interface,
```

**Review by RCheesley — APPROVED** [review]
GTG!

File: pages/28.Marketplace/default.en.md [hunk 18]
```diff
@@ -0,0 +1,138 @@
...
+
+The Marketplace is lacking some important features. In the text above there are mentions of several planned features in cursiva. Most of the features are waiting for Mautic 4 release and it's main features (Composer v2 support, new directory structure) and therefore can be developed when all these features will come together.
+
+- Mautic version compatibility check. The Marketplace should tell you whether a plugin is compatible with your Mautic installation.
+- Installation via CLI and UI. 
+- Update of installed plugins.
+- Automatic plugin updates. There will be configuration that will allow you to set globally whether you want to automatically upgrade plugins and also posibility to set that on per plugin bases. Automatic upgrades make sense only for bug fix releases. Other releases are too risky and should be handled manually. [API reference](https://packagist.org/apidoc#track-package-updates)
+- List security advisories. [API reference](https://packagist.org/apidoc#list-security-advisories)
+- Notifications about new versions and found security vulnerabilities.
+- Support also theme installations and updates.
```
[the remark this answers is not in the document]
Comment (RCheesley) [reply pages/28.Marketplace/default.en.md]: ^ This change was not committed in the PR. Adding another PR to add the link macros.

**Pull request merged**